pkg/rewards/claim.go

```diff
@@ -234,33 +372,37 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 				fmt.Println(calldataHex)
 			}
 		} else if config.OutputType == string(common.OutputType_Json) {
-			solidityClaim := claimgen.FormatProofForSolidity(accounts.Root(), claim)
-			jsonData, err := json.MarshalIndent(solidityClaim, "", "  ")
-			if err != nil {
-				logger.Error("Error marshaling JSON:", err)
-				return err
-			}
-			if !common.IsEmptyString(config.Output) {
-				err = common.WriteToFile(jsonData, config.Output)
+			for idx, claim := range *claims {
+				solidityClaim := claimgen.FormatProofForSolidity((*accounts)[idx].Root(), &claim)
+				jsonData, err := json.MarshalIndent(solidityClaim, "", "  ")
 				if err != nil {
+					logger.Error("Error marshaling JSON:", err)
```

Comment:
Small thing worth avoiding is logging an error and throwing the same error. Not such a big deal in CLIs, since you don't pay for logging, but something to keep in mind.

Reply:
yep. agreed. if you are throwing then no need to log.

Reply:
followed the format we had here: https://github.com/Layr-Labs/eigenlayer-cli/blob/5f6537f7cd9b9411d16abf17e878136cee7b4c4f/pkg/rewards/claim.go#L240

Removed the extra log